// 🔥 超簡単なtry/catchテスト

static box Main {
    init { console }
    
    main() {
        me.console = new ConsoleBox()
        me.console.log("🎯 try/catchテスト開始")
        
        // テスト1: 正常動作
        me.console.log("テスト1: 正常動作")
        try {
            me.console.log("try内部: 正常処理実行")
        } catch (Exception e) {
            me.console.log("catch: " + e)
        }
        me.console.log("テスト1完了")
        
        // テスト2: エラー発生
        me.console.log("テスト2: エラー発生")
        try {
            me.console.log("try内部: エラーを起こします")
            local fakeVar
            fakeVar = new NonExistentBox()
        } catch (Exception e) {
            me.console.log("catch成功: " + e)
        }
        me.console.log("テスト2完了")
        
        me.console.log("🎉 try/catchテスト終了")
        return "テスト成功"
    }
}